### PR TITLE
Fixed add-to-calendar incorrect event details issue on schedules page.

### PR DESCRIPTION
**Pull request by vamsiymca**

### CODE REVIEW

**Comment by vamsiymca** [comment]
this fix is related to this issue https://github.com/ynorth-projects/openy_repeat/issues/50

File: openy_system/assets/js/atc.dev.js [hunk 1]
```diff
@@ -316,10 +316,7 @@
           dom[tagnum].appendChild(atcb_list);
         } else {
           atcb_link = dom[tagnum].querySelector('.atcb-link');
-
-          if(dom[tagnum].querySelector('.atcb-list') == undefined) {
```
Comment (froboy) [inline openy_system/assets/js/atc.dev.js:320]: What is the implication of removing this conditional? Wouldn't this create two instances of `atcb_list` if the class already exists?

**Review by froboy — CHANGES_REQUESTED** [review]
@vamsiymca thanks for this fix. Could you please provide test steps for validating the fix, also steps to confirm that removing the conditional doesn't have negative implications?

Thanks.

File: openy_system/assets/js/atc.dev.js (hunk 1 above)
Comment (vamsiymca) [reply openy_system/assets/js/atc.dev.js:320]: @froboy  in line 308 , we are already checking if it is undefined or not

Comment (vamsiymca) [reply openy_system/assets/js/atc.dev.js:320]: @froboy  thank you for reviewing , if you click on on add-to-calendar and select google calendar sometimes it is taking previous event details for example change date to future date and click on any event you will see the previous date event detail https://www.nwsymca.org/group-exercise?date=2024-05-16&locations=&categories=&cn= , let me know if you have any questions.

Comment (vamsiymca) [reply openy_system/assets/js/atc.dev.js:320]: yeah you are right about that , sorry for confusion on that but this check is not in the main addtocalendarjs and i have tested possible scenarios from my side , it is not creating any new list and  is there any way we can test this one on sandbox using regression tests if we have any ?

Comment (froboy) [reply openy_system/assets/js/atc.dev.js:320]: Sounds good. Thanks for the explanation. Let's test it out.

**Review by froboy — APPROVED** [review]

File: openy_system/assets/js/atc.dev.js (hunk 1 above)
Comment (froboy) [reply openy_system/assets/js/atc.dev.js:320]: @vamsiymca please add your test steps to the main PR body, then @podarok can approve.

**Pull request merged**